Tests/test_pickle.py

```diff
@@ -92,3 +95,48 @@ def test_pickle_tell():
 
     # Assert
     assert unpickled_image.tell() == 0
+
+
+def helper_assert_pickled_font_images(font1, font2):
+    # Arrange
+    im1 = Image.new(mode="RGBA", size=(300, 100))
+    im2 = Image.new(mode="RGBA", size=(300, 100))
+    draw1 = ImageDraw.Draw(im1)
+    draw2 = ImageDraw.Draw(im2)
+    txt = "Hello World!"
+
+    # Act
+    draw1.text((10, 10), txt, font=font1)
+    draw2.text((10, 10), txt, font=font2)
+
+    # Assert
+    assert_image_equal(im1, im2)
+
+
+@pytest.mark.parametrize("protocol", list(range(0, pickle.HIGHEST_PROTOCOL + 1)))
```

Comment:
```suggestion
@pytest.mark.parametrize("protocol", range(0, pickle.HIGHEST_PROTOCOL + 1))
```
Testing, I find that `list` isn't needed.

Reply:
Yep, I did this originally but PyCharm's type checker complained:

![image](https://user-images.githubusercontent.com/1324225/141461187-bb2f6406-e54c-43d8-82cf-17289f8c81e6.png)

(`set` also silences it.)

Which do you prefer?